docs/Game/Unreal/getting-started/02-sending-transactions.md

```diff
@@ -0,0 +1,346 @@
+---
+title: 02 Sending Transactions & Retrieving Data
+id: unreal-send-retrieve
+---
+
+This page lists the different methods and functions you can make
+
+Send Transaction
+----------------
+----------------
+The **SendTransaction** function interacts with the blockchain and incurs gas fees. Signing in Metamask is required. 
```

Comment:
`SendTransaction` maybe? It's part of code, so shouldn't it be formatted as such?

Reply:
done